Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -1035,7 +1035,9 @@ Ref<MediaPromise> SourceBuffer::sourceBufferPrivateDidReceiveInitializationSegme
         // 5.6 Set first initialization segment flag to true.
         m_receivedFirstInitializationSegment = true;
 
+#if !USE(GSTREAMER)
```

Comment:
This code block whould also span to setMaximumBufferSize(), to avoid calling it on WPE platforms (please change to `!USE(WPE)`). Same for the call at line 187.

Reply:
Sure, I can make this change.